docs/guides/mongodb_cluster_okta_idp.md

```diff
@@ -111,52 +111,41 @@ locals {
       registry_key = "" # see container_registry_key in the downloaded template
     }
   }
-
-  # Specify the maximum number of nodes you expect this cluster to
-  # have, taking into consideration future growth. This number must be
-  # at least equal to the number of nodes currently in your
-  # cluster. This number is used for port reservation in the
-  # sidecar. This is the value that will be used for the `max_nodes`
-  # argument of the `properties` block in the repository resource (see
-  # resource `mongodb_repo` below).
-  mongodb_max_nodes = 5
-
-  # See `mongodb_port_alloc_range_low` and
-  # `mongodb_port_alloc_range_high` in the cyral_sidecar module
-  # configuration.
-  mongodb_ports_low  = 27017
-  mongodb_ports_high = local.mongodb_ports_low + local.mongodb_max_nodes
-
-  # All ports that will be used by MongoDB. This range must span at
-  # least the `local.mongodb_max_nodes` number of ports. Note that the
-  # port number you pass as the second argument to this function is
-  # not included in the range. For example, to set port 27021 as your
-  # uppermost port number, the second argument must be 27022.
-  mongodb_ports = range(local.mongodb_ports_low, local.mongodb_ports_high)
 }
 
 resource "cyral_repository" "mongodb_repo" {
   name = "mongodb_repo"
   type = "mongodb"
 
   # Specify the address or hostname of the endpoint of one node in the
-  # MongoDB replica set. Cyral will automatically/dynamically identify
-  # the remaining nodes of the replication cluster.
-  host = "mycluster-shard-00-01.example.mongodb.net"
+  # MongoDB replica set. You can explictly specify the host and port of
```

Comment:
@gengdahlCyral Can you review the comments on this guide? I need some guidance here to make sure that everything is explained properly :)